plan/task/nop.go

```diff
@@ -15,6 +15,6 @@ func init() {
 type nopTask struct {
 }
 
-func (t *nopTask) Run(_ context.Context, _ *plancontext.Context, _ *solver.Solver, v *compiler.Value) (*compiler.Value, error) {
-	return v, nil
+func (t *nopTask) Run(_ context.Context, _ *plancontext.Context, _ *solver.Solver, _ *compiler.Value) (*compiler.Value, error) {
+	return compiler.NewValue(), nil
```

Comment:
This won't return the input. You can lookup input and fill it into output.

Reply:
It's already done in CUE:

```cue
#Nop: {
	$dagger: task: _name: "Nop"
	input:  _
	output: input
}
```